data/transactions/logic/doc.go

```diff
@@ -101,16 +101,17 @@ var opDocByName = map[string]string{
 	"gitxna":  "Ith value of the array field F from the Tth transaction in the last inner group submitted",
 	"gitxnas": "Ath value of the array field F from the Tth transaction in the last inner group submitted",
 
-	"global":  "global field F",
-	"load":    "Ith scratch space value. All scratch spaces are 0 at program start.",
-	"store":   "store A to the Ith scratch space",
-	"loads":   "Ath scratch space value.  All scratch spaces are 0 at program start.",
-	"stores":  "store B to the Ath scratch space",
-	"gload":   "Ith scratch space value of the Tth transaction in the current group",
-	"gloads":  "Ith scratch space value of the Ath transaction in the current group",
-	"gloadss": "Bth scratch space value of the Ath transaction in the current group",
-	"gaid":    "ID of the asset or application created in the Tth transaction of the current group",
-	"gaids":   "ID of the asset or application created in the Ath transaction of the current group",
+	"global":   "global field F",
+	"load":     "Ith scratch space value. All scratch spaces are 0 at program start.",
+	"store":    "store A to the Ith scratch space",
+	"loads":    "Ath scratch space value.  All scratch spaces are 0 at program start.",
+	"stores":   "store B to the Ath scratch space",
+	"gload":    "Ith scratch space value of the Tth transaction in the current group",
+	"gloads":   "Ith scratch space value of the Ath transaction in the current group",
+	"gloadss":  "Bth scratch space value of the Ath transaction in the current group",
+	"gaid":     "ID of the asset or application created in the Tth transaction of the current group",
+	"gaids":    "ID of the asset or application created in the Ath transaction of the current group",
+	"json_ref": "return key B's value from a [valid](jsonspec.md) utf-8 encoded json object A",
```

Comment:
Can you put a blank line before "json_ref" and reformat?  It's logically quite different, and that way the diff will be a lot smaller, since the strings above won't get pushed right.

Reply:
👍